lib/ui/overflow_menu.dart

```diff
@@ -33,19 +33,31 @@ class _OverflowMenuState extends ConsumerState<OverflowMenu> {
 
   @override
   Widget build(BuildContext context) {
-    return PopupMenuButton<OverflowMenuOption>(
-      color: Theme.of(context).colorScheme.background,
-      icon: const Icon(Icons.more_vert),
-      shape: RoundedRectangleBorder(
-        side: const BorderSide(),
-        borderRadius: BorderRadius.circular(20),
+    return Theme(
```

Comment:
This could be stored in a style/theme file and make this file cleaner

Reply:
if we store this in a seprate file we woul need to store the related functions and so forth also there and the file would approximately contain the same things, or did you have a particular split in mind?